### ISSUE TITLE
Fail to build on Ubuntu 20 LTS [SOLVED]

### ISSUE THREAD

**Issue by dubigrasu** [stohrendorf/CroftEngine#290]
The process of installing what's needed and building went rather smooth except few warnings until 94%, when failed with:
`clang: error: linker command failed with exit code 1`
https://gist.github.com/dubigrasu/e8a9f8ae3e1dad93a1fa3073007f6a30

**Comment by stohrendorf** [comment]
Please don't use vcpkg on Linux, that's a Windows-only thing. I'll add that clarification to the readme.

**Comment by dubigrasu** [comment]
TL;DR: Build (with no vcpkg) has succeeded and the game works as expected.

The vcpkg bit has really threw me off, I imagined that is something like pip, and the fact that has a release for Linux added to my confusion. 
So far everything seems to work with two gamepads I've tested, feels good, looks good. Only tried it with the regular install from Steam, no other textures
![Screenshot from 2022-12-02 22-11-34](https://user-images.githubusercontent.com/6303599/205380217-517fbfcf-3b6b-47f2-93a4-825eeb76c642.png)

Thanks for the help and for the engine!
